src/app/pages/ArticlePage/ArticlePage.tsx

```diff
@@ -288,6 +289,9 @@ const ArticlePage = ({ pageData }: { pageData: Article }) => {
         experimentVariant,
       ),
   );
+
+  const readTime = pageData?.metadata?.stats?.readTime || undefined;
```

Comment:
This is not really needed - if there's no readTime in the data, then it will be undefined bby default
```suggestion
  const readTime = pageData?.metadata?.stats?.readTime;
```